src/equations/equations.jl

```diff
@@ -20,6 +20,16 @@ end
 @inline Base.ndims(::AbstractEquations{NDIMS}) where NDIMS = NDIMS
 
 
+"""
+    calcflux(u, orientation, equations)
+
+Calculate the flux in space direction `orientation` associated to the
+`equations` given the conservative variables `u`.
```

Comment:
Maybe to match the sorting and to give slightly more context:
```suggestion
Given the conservative variables `u`, calculate the (physical) flux in spatial direction `orientation` for the coressponding set of governing `equations` . `orientation` is `1`, `2`, and `3` for the x-, y, and z-directions.
```

Reply:
:+1: